git-gutter.el

```diff
@@ -664,12 +664,14 @@ gutter information of other windows."
                     index)))
 
 (defun git-gutter:search-here-diffinfo (diffinfos)
-  (cl-loop with current-line = (line-number-at-pos)
-           for diffinfo in diffinfos
-           for start = (plist-get diffinfo :start-line)
-           for end   = (or (plist-get diffinfo :end-line) (1+ start))
-           when (and (>= current-line start) (<= current-line end))
-           return diffinfo))
+  (save-excursion
+    (widen)
```

Comment:
widen needs save-restriction, not save-excursion, otherwise, user's narrowing will be removed.